api/src/devcomp/domain/usecases/delete-training-trigger.js

```diff
@@ -1,6 +1,21 @@
 import { withTransaction } from '../../../shared/domain/DomainTransaction.js';
+import { NotFoundError } from '../../../shared/domain/errors.js';
+
+const deleteTrainingTrigger = withTransaction(async function ({
+  trainingId,
+  trainingTriggerId,
+  trainingTriggerRepository,
+}) {
+  const trainingTriggers = await trainingTriggerRepository.findByTrainingId({
+    trainingId,
+  });
+
+  const trainingTriggerToDelete = trainingTriggers.find((trainingTrigger) => trainingTrigger.id === trainingTriggerId);
+
+  if (!trainingTriggerToDelete) {
```

Comment:
Tu peux remplacer le `.find` par un `.some` car on veut juste savoir si le déclencheur concerné  appartient au CF 
```suggestion
const isTrainingTriggerRelatedToTraining = trainingTriggers.some((trainingTrigger) => trainingTrigger.id === trainingTriggerId);

  if (!isTrainingTriggerRelatedToTraining) {
```